examples/basics/emag_basics.py

```diff
@@ -3,220 +3,249 @@
-# Get torque and voltage data.
-shaft_torque = mcad.get_variable("ShaftTorque")
-line_voltage = mcad.get_variable("PeakLineLineVoltage")
+# Retrieve the torque and voltage output data values.
+shaft_torque = mc.get_variable("ShaftTorque")
+line_voltage = mc.get_variable("PeakLineLineVoltage")
+print(
+    f"Shaft Torque: {shaft_torque:.2f} Nm, Line-Line Terminal Voltage (peak): {line_voltage:.2f} V"
+)
 
 # %%
-# Graph the torque data.
-num_torque_points = points_per_cycle * number_cycles
-rotor_position = []
-torque_vw = []
-
-for n in range(num_torque_points):
-    (x, y) = mcad.get_magnetic_graph_point("TorqueVW", n)
-    rotor_position.append(x)
-    torque_vw.append(y)
+# Retrieve the *Torque (VW)* graph data. The data type for this series is *E-Magnetics*, so use the
+# ``get_magnetic_graph()`` method.
+#
+# .. note::
+#    When retrieving graph data points, use the *Graph Viewer* under *Help* in the Motor-CAD
+#    interface to find the series names and data types.
+rotor_position, torque_vw = mc.get_magnetic_graph("TorqueVW")
 
 # %%
-# Graph the airgap flux density data.
-loop = 0
-success = 0
-mech_angle = []
-airgap_flux_density = []
+# Retrieve the airgap flux density graph data. This data can be found in the *Graph Viewer* in
+# Motor-CAD. The data type for this series is *FEA Path*, so use the ``get_fea_graph()`` method.
+mech_angle, airgap_flux_density = mc.get_fea_graph("B Gap (on load)", 1, 0)
 
 # %%
-# Keep looking until you cannot find the point.
-while success == 0:
-    try:
-        (x, y) = mcad.get_fea_graph_point("B Gap (on load)", 1, loop, 0)
-        mech_angle.append(x)
-        airgap_flux_density.append(y)
-        loop = loop + 1
-    except pymotorcad.MotorCADError:
-        success = 1
+# Only the most recently displayed harmonic graphs are available in Motor-CAD via the
+# *Graph Viewer*. To retrieve harmonic data by using Motor-CAD automation, first display the
+# relevant *Graphs -> Harmonics* tab. Use the ``initialise_tab_names()`` method to initialise the
+# available tabs in the Motor-CAD user interface. When the tabs have been initialised, the
+# ``display_screen()`` method can be used until either the Motor-CAD context is changed, or the
+# instance is closed.
+#
+# Open the *Graphs -> Harmonics -> Torque* tab to make the torque harmonic data available.
+mc.initialise_tab_names()
+mc.display_screen("Graphs;Harmonics;Torque")
 
 # %%
-# Graph the harmonic data.
-mcad.initialise_tab_names()
-mcad.display_screen("Graphs;Harmonics;Torque")
-
-num_harmonic_points = (points_per_cycle * number_cycles) + 1
-data_point = []
-torque = []
-for n in range(num_harmonic_points):
-    try:
-        (x, y) = mcad.get_magnetic_graph_point("HarmonicDataCycle", n)
-        data_point.append(x)
-        torque.append(y)
-    except pymotorcad.MotorCADError:
-        print("Results failed to export.")
-
+# Retrieve the torque harmonic graph data. The data type for this series is *E-Magnetics*, so use
```

Comment:
```suggestion
# Retrieve the torque harmonic graph data. The data type for this series is **E-Magnetics**, so use
```
Double asterisk for bold for UI element, but double backticks if this is a code entity.